public/embed.js

```diff
@@ -0,0 +1,144 @@
+"use strict";
```

Comment:
Why is in the public folder? Is webpack fetching it in some way? Why don't just move them to lib/commons?